packages/snaps-controllers/src/snaps/SnapController.test.ts

```diff
@@ -2377,18 +2377,19 @@ describe('SnapController', () => {
         controller.installSnaps(MOCK_ORIGIN, {
           [snapId]: {},
         }),
-      ).rejects.toThrow('Invalid snap id. Unknown prefix. Received: "foo".');
+      ).rejects.toThrow('Invalid snap ID:');
     });
 
     it('returns an error if an origin does not have the permission to install a snap', async () => {
       const controller = getSnapController();
 
       await expect(
         controller.installSnaps(MOCK_ORIGIN, {
-          bar: {},
+          // eslint-disable-next-line @typescript-eslint/naming-convention
+          'npm:bar': {},
```

Comment:
Can we use `MOCK_SNAP_ID` instead?

Reply:
It requires some changes to the test, but I can make those